dotnet-docs/azure-marketplace-integration.md

```diff
@@ -0,0 +1,80 @@
+
+	![Configuring the Basics section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-basics.png)
+
+1. Configure the __Virtual Machine__ section. The screenshot below shows the recommended _Ubuntu OS Version_, _CPU Architecture_, and _VM size_:
+
+	![Configuring the Virtual Machine section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-virtual-machine.png)
+
+	You may use an existing IP Address, or create a new one with the button at the bottom of the _Public IP Address for the VM_ setting.
+
+	>important The __DNS Prefix for the public IP Address__ will be concatenated with the string below it to form the Report Server URL, where you will be able to access the Report Server Manager.
+
+1. Add the __License__ token for your Telerik Report Server product - see how to set up the [Telerik Report Server License]({%slug license-key%}):
+
+	![Configuring the License section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-license.png)
+
+1. Configure the __Administrator__ settings:
+
+	![Configuring the Administrator section of the Report Server .NET resource on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-administrator.png)
+
+1. The __Mail__ settings are optional. You may configure the SMTP in the Report Server configuration after the deployment.
+1. The __Storage__ section lets you select the MSSQL Database storage. The default Storage type is File Storage.
+1. In __HTTPS__, you may choose a Free HTTPS Certificate.
+1. The last section __Review + create__ validates the above settings.
+
+	![Review the Report Server .NET resource settings on Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-settings-review.png)
+
+	After the validation passes, you need to click on the `Create` button to start the deployment of your Report Server .NET.
+
+1. Wait for the deployment to finish. You will see a screen like below during the process that may take several minutes:
+
+	![The deployment of the Report Server .NET on Azure Marketplace is in progress.](../images/rs-net-images/rs-net-azure-marketplace-settings-deployment-in-progress.png)
+
+	When all the items under the _Resource_ are marked with green ticks, the deployment has finished successfully, and you will see the next screen:
+
+	![The deployment of the Report Server .NET on Azure Marketplace has finished successfully.](../images/rs-net-images/rs-net-azure-marketplace-settings-deployment-complete.png)
+
+1. Access your Report Server on the URL specified in the _Virtual Machine_ setting:
+
+	![Report Server .NET Manager log in screen after successful deployment from the Azure Marketplace.](../images/rs-net-images/rs-net-azure-marketplace-access-rs.png)
+
+## See Also
+
```

Comment:
Maybe it's a good idea to place a troubleshooting section. If the installation takes too much time after the setup gets to the SetupReportServer step, they need to click on the VM line from the list with the installation progress. Then see if there is a warning message at the top telling "<The-name-of-the-vm> virtual machine agent status is not ready. Troubleshoot the issue". If it is there after several minutes waiting on the last step of the installation, then maybe they need to stop the installation. Delete all the resources from their resource group and start the process again.
![image](https://github.com/user-attachments/assets/1aa87dac-cfa3-416a-8b30-970336f07163)

Reply:
Thank you, we will create a dedicated article (KB) with all the screenshots we created.